### PR TITLE
Raidboss: Shiva hm/ex + common

### PR DESCRIPTION
**Pull request by VanSlanzar**
- adds missing translations fr.
- removal lines `adds targetable` on shiva-ex/hm + epic_alex for add common. there was a misunderstanding between appear and targetable on the German translation that I corrected.  anvisierbar=targetable and erscheinen=appear

### CODE REVIEW

File: ui/raidboss/common_replacement.js [hunk 1]
```diff
@@ -44,6 +44,13 @@ const commonReplacement = {
       cn: '--小怪出现--',
       ko: '--쫄 소환--',
     },
+    '--adds targetable--': {
+      de: '--adds anvisierbar--',
+      fr: '--adds ciblables--',
+      ja: '--fixme--',
```
Comment (VanSlanzar) [inline ui/raidboss/common_replacement.js:50]: @MaikoTan can you help me for this translate thank you.

**Comment by quisquous** [comment]
Do you want to wait for @MaikoTan or do you want me to merge this now and follow up with Japanese changes later?

File: ui/raidboss/common_replacement.js (hunk 1 above)
Comment (MaikoTan) [reply ui/raidboss/common_replacement.js:50]: `ja: '雑魚ターゲット可能',`

Comment (VanSlanzar) [reply ui/raidboss/common_replacement.js]: Thank you.

**Comment by VanSlanzar** [comment]
Ok you can to merge.

**Comment by quisquous** [comment]
Thanks!

**Pull request merged**